Python/bytecodes.c

```diff
@@ -3977,11 +3930,14 @@ dummy_func(
         // stack effect: ( -- )
         inst(EXTENDED_ARG) {
             assert(oparg);
+            opcode = _Py_OPCODE(*next_instr);
+            if (cframe.use_tracing) {
```

Comment:
We could handle `EXTENDED_ARG` in `DO_TRACING`. It makes `DO_TRACING` even slower, but we can then assert `cframe.use_tracing == 0` here.